src/components/news_item/NewsItem.tsx

```diff
@@ -1,16 +1,23 @@
 import * as React from 'react';
+import { urlFor } from '../../core/utilities/image-builder.functions';
 
 export default function NewsItem(props: any) {
+  
+  const logoUrl = urlFor(props.news.logo).url();
 
   return (
-    <div className="newsitem">
-      <div className="newsitem__section">
-        <img className="newsitem__logo" src={props.logoUrl}/>
+  <li key={props.news.key} className="newsItem newsItem__list-item">
+      <div className="newsItem__section">
+        <a href={props.news.url}>
```

Comment:
Lets add target="_blank" https://html.com/attributes/a-target/.
I will make a task for Marina to design a way to let users know it jumps to a new page